src/osd/scrubber/pg_scrubber.cc

```diff
@@ -537,6 +537,16 @@ void PgScrubber::on_primary_change(
     << dendl;
 }
 
+/*
+ * A note re the call to publish_stats_to_osd() below:
```

Comment:
How about rephrasing this lline? Run into a _parser error_.

Reply:
@rzarzynski - not sure I understand the remark?

Reply:
Nothing big -- today it was pretty parseable. I think last time it was the combination of singular (_a note_), _re_ and lack of verb before _below_.